.github/workflows/autoformat.yml

```diff
@@ -115,8 +115,8 @@ jobs:
       # ************************************************************************
       - if: steps.diff.outputs.python == 'true'
         run: |
-          ruff --fix .
-          black .
+          ruff check --fix .
+          ruff format .
```

Comment:
Should we disable those lint rules when using format https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules

Reply:
I don't think we enable these conflicting rules

Reply:
If we did, ruff would warn us but I don't see any warnings.